bentoml/_internal/server/http_app.py

```diff
@@ -264,6 +264,8 @@ def on_startup(self) -> list[t.Callable[[], None]]:
         else:
             for runner in self.bento_service.runners:
                 on_startup.append(runner.init_client)
+        for runner in self.bento_service.runners:
+            asyncio.get_event_loop().run_until_complete(runner.runner_handle_is_ready())
```

Comment:
This doesn't actually check the result of the `is_ready` call, it just runs it.

You'll probably want to have a separate async function to call this in a loop instead; maybe using `asyncio.gather` to run all the requests to the runners in parallel.

Reply:
Thanks again. I fixed this section in the latest commits.

Reply:
👍